.github/workflows/docker-mce-consumer.yml

```diff
@@ -12,10 +12,21 @@ jobs:
     if: ${{ github.repository == 'linkedin/datahub' }}
     steps:
       - uses: actions/checkout@v2
+      - id: tag
+        run: |
+          TAG=$(echo ${GITHUB_REF} | sed -e 's/refs\/heads\/master/latest/g' | sed -e 's/refs\/tags\///g')
+          echo "tag=$TAG"
+          echo "::set-output name=tag::$TAG"
       - uses: docker/build-push-action@v1
         with:
           dockerfile: ./docker/mce-consumer/Dockerfile
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
           repository: linkedin/datahub-mce-consumer
-          tag_with_ref: true
+          tags: ${{ steps.tag.outputs.tag }}
+      - uses: VaultVulp/gp-docker-action@1.1.6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          dockerfile: ./docker/gms/Dockerfile
```

Comment:
This seems wrong.

Reply:
Can you more details to these comments? Why is it wrong?

Reply:
Wrong dockerfile. Will fix that.